features/service-operation.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Command output from service operations like stop/start/restart
+  service-manager should return the correct exit code from stop/start/restart command
+
+  @operation
+  Scenario Outline: Boot and execute service operations like stop/start/restart
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    require 'vagrant-libvirt'
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+    end
+    """
+
+    When I successfully run `bundle exec vagrant up --provider <provider>`
+      And I successfully run `bundle exec vagrant service-manager stop docker`
```

Comment:
These tests are "verifying" the existence of the start/stop/restart command which is good, but they don't check at all, if they work at all. We should actually verify the state on the VM as well and check that the commands do what we want. 

You can run commands on the VM like this:

    run("vagrant ssh -c \"systemd status docker\"")
    expect(last_command_started).to have_exit_status(0)
    expect(last_command_started).to have_output(/<whatver makes sense to match in order to verify that the service is started/stopped/restarted>/)

See also https://github.com/projectatomic/adb-vagrant-registration/pull/70/files